x-pack/plugins/translations/translations/en.json

```diff
@@ -145,7 +143,7 @@
     'xpack.enterpriseSearch.content.indices.configurationConnector.researchConfiguration.serviceDocumentationLinkLabel': '{name} documentation',
     'xpack.enterpriseSearch.content.indices.configurationConnector.scheduleSync.description': 'Once your connectors are configured to your liking, don’t forget to set a recurring sync schedule to make sure your documents are indexed and relevant. You can also trigger a one-time sync without enabling a sync schedule.',
     'xpack.enterpriseSearch.content.indices.configurationConnector.steps.deployConnector.title': 'Deploy a connector',
-    'xpack.enterpriseSearch.content.indices.configurationConnector.steps.enhance.title': 'Enhance your connector client',
+    'xpack.enterpriseSearch.content.indices.configurationConnector.steps.enhance.title': 'Configure your connector client',
```

Comment:
We generally don't want to manually adjust copy in these files outside of deleting unused lines. The new copy will be automatically picked up down the line.

Reply:
Aha! Yes I did that manually before remembering to use the cli tool